docs/network-flow-visibility.md

```diff
@@ -387,11 +387,36 @@ and throughput is shown in the line graphs.
 #### Network-Policy Flows Dashboard
 
 Network-Policy Flows Dashboard visualizes the traffic with NetworkPolicies enforced.
-Currently we only support the visualization of NetworkPolicies with `Allow` action.
+The Chord diagram visualizes the cumulative bytes of all the traffic in the selected
+time range. We use green color ![#228B22](https://via.placeholder.com/15/228B22/000000?text=+)
+to highlight traffic with "Allow" NetworkPolicy enforced, red color
+![#EE4B2B](https://via.placeholder.com/15/EE4B2B/000000?text=+) to highlight
+traffic with "Deny" NetworkPolicy enforced. Unprotected traffic without NetworkPolicy
+enforced has the same color with its source Pod. Every link is in the shape of
+an arrow, pointing from source to destination. Line graphs shows the evolution
+of traffic throughput with NetworkPolicy enforced.
 
-<img src="https://downloads.antrea.io/static/02152022/flow-visibility-np-1.png" width="900" alt="Network-Policy Flows Dashboard">
+<img src="https://github.com/heanlan/grafana-chord-plugin/releases/download/1.0.0/np-0.png" width="900" alt="Network-Policy Flows Dashboard">
```

Comment:
shouldn't those images be stored under antrea/theia repo (same below)?

Reply:
all of the generated links are temporary. I'll upload those to antrea.io S3 once we finalize the PR.